### ISSUE TITLE
Bird Call Classification Datasets (Birdcalls71)

### ISSUE THREAD

**Issue by oliver-adams-b** [earthspecies/library#17]
[This](https://figshare.com/s/4af71d71d94e04afcd5f) dataset contains audio recordings of 71 different bird species gathered from three different sources ([The Macaulay Library](http://www.macaulaylibrary.org), the [Art & Science Centre (UCLA)](http://artsci.ucla.edu/birds/database.html), and the [Great Himalayan national Park dataset](https://www.greathimalayannationalpark.org/biodiveristy/fauna/)). The full dataset is only ~450MB, and is split up by folder into testing, training, and validation sets.  The original recording lengths vary from 0.5 seconds to 320 seconds, and are all sampled at 44.1kHz. However, in the provided dataset only the Mel-spectrograms of the bird calls are provided, and are stored in `.npy` format. 

The Birdcalls71 dataset was found in the paper (?) by Anshul Thakur et al:  ["**Multiscale CNN based Deep Metric Learning for Bioacoustic Classification: Overcoming Training Data Scarcity Using Dynamic Triplet Loss**"](https://arxiv.org/abs/1903.10713) (still under review) (I suspect the researchers are the people who curated this dataset, though I'm not completely sure).

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1198183573587004/1198566177751088) by [Unito](https://www.unito.io/learn-more)

**Comment by radekosmulski** [comment]
Closing - the dataset is interesting but we don't want to include in our library a dataset that starts with spectrograms, we would like to start with audio.
